PowerUp/.idea/misc.xml

```diff
@@ -1,5 +1,32 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="EntryPointsManager">
```

Comment:
Same here: this file is configuration for Android Studio and should not be commited